### PR TITLE
check-in

### PR DESCRIPTION
**Pull request by bhumulanandinireddy**
Added check-in functionality
Both frontend and backend
Added test cases(all passed)

### CODE REVIEW

**Review by BaloneyBoy97 — COMMENTED** [review]
back end server logic is sound.
Suggestion:
- create test cases before merging check in module
- add flask mail to send email notification to use to confirm check in
- in check_in function create a mock plane ticket for user to board the air plane.

**Comment by VennapusalaCharitha** [comment]
went through the code and testcases the code seems fine and is following other feature module format. could you confirm if the overall app and the flow still works the same.

**Comment by bhumulanandinireddy** [comment]
> ollowing other feature module format

yes

**Comment by bhumulanandinireddy** [comment]
> went through the code and testcases the code seems fine and is following other feature module format. could you confirm if the overall app and the flow still works the same.

yes

**Comment by VennapusalaCharitha** [comment]
ok, im approving it.

File: airView/checkin/feature.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from flask import Blueprint, request, jsonify
```
Comment (VennapusalaCharitha) [inline airView/checkin/feature.py:1]: used blueprint (follows the other modules)

**Review by VennapusalaCharitha — APPROVED** [review]

**Comment by bhumulanandinireddy** [comment]
> back end server logic is sound. Suggestion:
> 
> * add flask mail to send email notification to use to confirm check in
> * in check_in function create a mock plane ticket for user to board the air plane.

added the code for checkin flask email confirmation

**Pull request merged**